lib/c_glib/src/thrift/c_glib/transport/thrift_memory_buffer.c

```diff
@@ -209,16 +242,39 @@ thrift_memory_buffer_class_init (ThriftMemoryBufferClass *cls)
 
   param_spec = g_param_spec_uint ("buf_size",
                                   "buffer size (construct)",
-                                  "Set the read buffer size",
+                                  "Set the read/write buffer size limit"
+                                  " (0 for unlimited)",
                                   0, /* min */
```

Comment:
` " (0 for unlimited)",  0, /* min */` is confusing. Is the minimum actually unlimited ?